hack/generator/pkg/codegen/pipeline/testdata/TestCreateStorageTypes-v20211231.golden

```diff
@@ -3,10 +3,20 @@
 // Licensed under the MIT license.
 package v20211231
```

Comment:
Should the package have changed? Or is the destination package an input to the code that this test is checking?

Reply:
I'll double check this after moving the graph creation code; when I reviewed these changes prior to creating the PR I was sure they were correct, but I'm no longer convinced.

Reply:
They have no longer changed - I fixed a couple of bugs in the test. Good catch.